aim/aim_lib/nat_strategy.py

```diff
@@ -135,7 +135,7 @@ def get_l3outside_resources(self, ctx, l3outside, epg_name=None):
         """
 
     @abc.abstractmethod
-    def create_subnet(self, ctx, l3outside, gw_ip_mask):
+    def create_subnet(self, ctx, l3outside, gw_ip_mask, scope):
```

Comment:
The abstract base class methods. should have the same signature as their concrete class methods.